Assets/Scripts/PuzzleSwitch.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
```

Comment:
I am guessing this can be deleted? I noticed the other file SwitchPuzzle seems to have the actual logic.

Reply:
I think this one is for the levers. I think I need to rename it.